core/exec.c

```diff
@@ -963,7 +963,59 @@ static lai_api_error_t lai_exec_reduce_op(int opcode, lai_state_t *state, struct
         lai_operand_emplace(state, &operands[2], &result);
         break;
     }
+    case MID_OP: {
+        lai_variable_t object = {0};
```

Comment:
lai_variable_t's should be created with `LAI_CLEANUP_VAR lai_variable_t obj = LAI_VAR_INITIALIZER;`

Reply:
I probably got confused by how is created in some other places of LAI 😅